test/mpi/pt2pt/testlist

```diff
@@ -49,3 +49,5 @@ huge_anysrc 2
 dtype_send 2
 recv_any 2
 irecv_any 2
+dtpools_simple 2
+dtpools_full 2
```

Comment:
These are only temporary, they will be removed eventually and DTPools functionalities integrated in current datatype tests.